pkgs/games/osu/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, appimageTools, fetchurl, icu }:
+
+let
+  pname = "osu";
+  version = "2020.408.1";
+  sha256 = "15znj9ka0zqc24fhmvxp7vxf35i29n8f0sdkgn9wplal79ycjkgg";
+
+in appimageTools.wrapType2 rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/ppy/osu/releases/download/${version}/osu.AppImage";
+    inherit sha256;
+  };
+
+  extraInstallCommands = "mv $out/bin/{${name},${pname}}";
+
+  profile = ''
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH''${LD_LIBRARY_PATH:+:}${icu}/lib"
+  '';
+
+  meta = with lib; {
+    description = "Rhythm is just a click away";
+    homepage = "https://osu.ppy.sh";
+    license = licenses.mit;
+    maintainers = with maintainers; [ oxalicc ];
```

```suggestion
    maintainers = with maintainers; [ oxalica ];
```

One letter off, and borg would have been happy :P

---

Oops, it's not evaluated when building. :(